css-ui-4/Overview.bs

```diff
@@ -93,6 +93,10 @@ spec:css-color-4; type:property; text:color
 spec:selectors-4; type:selector; text::focus
 spec:selectors-4; type:selector; text::focus-visible
 </pre>
+<pre class=anchors>
+urlPrefix: https://html.spec.whatwg.org/multipage/; spec:HTML
+	text:password; type:attr-value; for:input/type; url: input.html#attr-input-type-password-keyword
+</pre>
```

Comment:
I understand that the HTML markup is bad here and you *can't* reasonably use link-defaults, so this is fine for now. When you do the HTML PR, could you submit a fixup for these dfns as well?

Reply:
Will do!